### PR TITLE
Added pr2_head node from natural_language branch.

### PR DESCRIPTION
**Pull request by 23pointsNorth**
---

This gives the functionality to move the head to a desired frame as well as to
use the audio capabilities of the robot.

Added timeouts and warnings for the action servers.

### CODE REVIEW

File: pr2_head/src/head.cpp [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  this->lookAt("base_link", table_pos_.x, table_pos_.y, table_pos_.z);
+}
+
+void Head::updateLookingPosition() {
+  bool new_obj = target_object_ != look_at_object_;
+  if (target_object_ != "") {
+    if ((new_obj) || (follow_object_ == true) ) {
+      look_at_object_ = target_object_;
+      if (new_obj) {
+        // std::string sentence = "Looking at " + look_at_object_;
```
Comment (GreatAlexander) [inline pr2_head/src/head.cpp]: Could you add a ROS param to determine if this commented code gets executed?
Like: if (new_obj && declare_obj) {
UNCOMMENTED LINES}

**Comment by GreatAlexander** [comment]
Looks good! Could you do a tiny change so we don't have wasted commented code I left behind?

**Comment by 23pointsNorth** [comment]
I've done that edit. + added services for yap-ing and nay-ing.

**Comment by GreatAlexander** [comment]
You got it! I saw you pulled it up to 10 Hz now, and if everything still works, then go ahead and merge! +1

**Comment by 23pointsNorth** [comment]
I've tested it only with moving the head wrt base_link. Is it purposefully low at 5? Haven't noticed anything whilst moving the head.

**Comment by GreatAlexander** [comment]
I thought 5 Hz was good enough, it should affect tracking an object while
moving about. Try that with 10 Hz, it should still work just as well.

**Comment by 23pointsNorth** [comment]
It's tracking the "milk" fine (a.k.a. as fast as the detector streams). :)

P.S. Fix for future release is to nod/shake in the direction that it is already looking at.

**Pull request merged**